doc/man3/OSSL_HTTP_transfer.pod

```diff
@@ -218,7 +219,8 @@ other HTTP client implementations such as wget, curl, and git.
 =head1 RETURN VALUES
 
 OSSL_HTTP_get(), OSSL_HTTP_get_asn1(), OSSL_HTTP_post_asn1(), and
-OSSL_HTTP_transfer() return on success the data received via HTTP, else NULL.
+OSSL_HTTP_transfer() return on success a memory BIO containing
```

Comment:
How about: _On success, OSSL_HTTP_transfer() returns a memory BIO containing the data received via HTTP.  This must be freed by the caller.  On failure, NULL is returned._